python/py-terminaltables/Portfile

```diff
@@ -24,7 +25,17 @@ checksums           rmd160  9d02523f89d32c890ae9c40212155e221584e279 \
                     size    12264
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
+    patch.pre_args  -p1
+    patchfiles      10-fix-poetry.patch
+
+    pre-patch {
+        # Convert files from DOS to UNIX linefeeds before patching.
+        reinplace "s/\r$//g" ${worksrcpath}/pyproject.toml
+    }
+
+    depends_build-append \
+                    port:py${python.version}-setuptools \
```

Comment:
are you sure that `setuptools` is still needed if the port uses the `poetry-core` backend?

Reply:
Not really, lemme fix that.